### PR TITLE
Solve the problem of garbled file names when downloading Chinese files

### PR DESCRIPTION
**Pull request by mulzhang**
Add file name coding to solve the problem of downloading Chinese file names

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

**Comment by mulzhang** [comment]
Import required "net/url"

**Comment by mulzhang** [comment]
@harshavardhana， Please enter the merge code

File: restapi/user_objects.go [hunk 1]
```diff
@@ -415,7 +416,7 @@ func getDownloadObjectResponse(session *models.Principal, params user_api.Downlo
 			rw.Header().Set("X-Frame-Options", "SAMEORIGIN")
 			rw.Header().Set("X-XSS-Protection", "1")
 		} else {
-			rw.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", filename))
+			rw.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", url.QueryEscape(filename)))
```
Comment (harshavardhana) [inline restapi/user_objects.go:419]: This should be url.PathEscape

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by harshavardhana** [comment]
Fixed by #1366

**Pull request closed without merging**